src/Orchard.Web/Core/Contents/Controllers/AdminController.cs

```diff
@@ -394,31 +403,27 @@ public class AdminController : ContentControllerBase, IUpdateModel {
 
         [HttpPost]
         public ActionResult Clone(int id, string returnUrl) {
-            var contentItem = _contentManager.GetLatest(id);
-
-            if (contentItem == null)
-                return HttpNotFound();
+            var originalContentItem = _contentManager.GetLatest(id);
 
-            if (!Services.Authorizer.Authorize(Permissions.CreateContent, contentItem, T("Couldn't clone content")))
+            if (!Services.Authorizer.Authorize(Permissions.ViewContent, originalContentItem, T("Couldn't open original content")))
```

Comment:
Can you explain why these permissions are changed?

Reply:
We started from a fork version of LaserOrchard which already has these changes, it is a part of code that I did not write but which I had to integrate to remain aligned with the version we started from. Ours mod is just a proposal, which unfortunately integrates changes that were not made by us, if too different from the code, as far as I'm concerned it can be rejected entirely

Reply:
These changes are from the dev branch.